src/BlazorWebView/src/Maui/iOS/IOSWebViewManager.cs

```diff
@@ -123,10 +124,9 @@ public override void RunJavaScriptConfirmPanel(WKWebView webView, string message
 					cancelAction: _ => completionHandler(false)
 				);
 			}
-
-#pragma warning disable CS0672 // Member overrides obsolete member
-			public override void RunJavaScriptTextInputPanel(
-#pragma warning restore CS0672 // Member overrides obsolete member
+			
+			[Export("webView(_:runJavaScriptTextInputPanelWithPrompt:defaultText:initiatedByFrame:completionHandler:)")]
+			public void RunJavaScriptTextInputPanelWithPrompt(
```

Comment:
in .NET iOS we use: https://github.com/dotnet/macios/blob/29de197d989052903d126581cb768f107d256931/src/webkit.cs#L5032
that is using the deprecated method: https://developer.apple.com/documentation/webkit/webuidelegate/webview(_:runjavascripttextinputpanelwithprompt:defaulttext:initiatedby:)

The new way to go is use: https://developer.apple.com/documentation/webkit/wkuidelegate/webview(_:runjavascripttextinputpanelwithprompt:defaulttext:initiatedbyframe:completionhandler:)

Reply:
I don't see that in .NET 10 preview 5...

Reply:
Until XAMCORE 5 is ready, we need this workaround. :(